PhysicsTools/NanoAOD/python/taus_updatedMVAIds_cff.py

```diff
@@ -355,194 +359,124 @@
-    mapping = cms.VPSet(
-        cms.PSet(
-            category = cms.uint32(0),
-            cut = cms.string('RecoTauTag_antiElectron'+antiElectronDiscrMVA6_version+'_gbr_NoEleMatch_woGwoGSF_BL_WPeff70'),
-            variable = cms.string('pt')
-        ),
-        cms.PSet(
-            category = cms.uint32(2),
-            cut = cms.string('RecoTauTag_antiElectron'+antiElectronDiscrMVA6_version+'_gbr_NoEleMatch_wGwoGSF_BL_WPeff70'),
-            variable = cms.string('pt')
-        ),
-        cms.PSet(
-            category = cms.uint32(5),
-            cut = cms.string('RecoTauTag_antiElectron'+antiElectronDiscrMVA6_version+'_gbr_woGwGSF_BL_WPeff70'),
-            variable = cms.string('pt')
-        ),
-        cms.PSet(
-            category = cms.uint32(7),
-            cut = cms.string('RecoTauTag_antiElectron'+antiElectronDiscrMVA6_version+'_gbr_wGwGSF_BL_WPeff70'),
-            variable = cms.string('pt')
-        ),
-        cms.PSet(
-            category = cms.uint32(8),
-            cut = cms.string('RecoTauTag_antiElectron'+antiElectronDiscrMVA6_version+'_gbr_NoEleMatch_woGwoGSF_EC_WPeff70'),
-            variable = cms.string('pt')
-        ),
-        cms.PSet(
-            category = cms.uint32(10),
-            cut = cms.string('RecoTauTag_antiElectron'+antiElectronDiscrMVA6_version+'_gbr_NoEleMatch_wGwoGSF_EC_WPeff70'),
-            variable = cms.string('pt')
-        ),
-        cms.PSet(
-            category = cms.uint32(13),
-            cut = cms.string('RecoTauTag_antiElectron'+antiElectronDiscrMVA6_version+'_gbr_woGwGSF_EC_WPeff70'),
-            variable = cms.string('pt')
-        ),
-        cms.PSet(
-            category = cms.uint32(15),
-            cut = cms.string('RecoTauTag_antiElectron'+antiElectronDiscrMVA6_version+'_gbr_wGwGSF_EC_WPeff70'),
-            variable = cms.string('pt')
-        )
-    )
-)
+patTauDiscriminationByTightElectronRejectionMVA62018 = patTauDiscriminationByVLooseElectronRejectionMVA62018.clone()
+for m in patTauDiscriminationByTightElectronRejectionMVA62018.mapping:
+    m.cut = m.cut.value().replace("WPeff98", "WPeff70")
 # VTight
-patTauDiscriminationByVTightElectronRejectionMVA62018 = patTauDiscriminationByVLooseElectronRejectionMVA62018.clone(
+patTauDiscriminationByVTightElectronRejectionMVA62018 = patTauDiscriminationByVLooseElectronRejectionMVA62018.clone()
+for m in patTauDiscriminationByVTightElectronRejectionMVA62018.mapping:
+    m.cut = m.cut.value().replace("WPeff98", "WPeff60")
+### Define v1 anit-e discriminants (2015)
+antiElectronDiscrMVA6_version = "MVA6v1"
+## Raw
+patTauDiscriminationByElectronRejectionMVA62015Raw = patTauDiscriminationAgainstElectronMVA6.clone(
+    Prediscriminants = noPrediscriminants, #already selected for MiniAOD
+    vetoEcalCracks = True, #don't keep tau candidates in EB-EE cracks for v1
+    mvaName_NoEleMatch_wGwoGSF_BL = 'RecoTauTag_antiElectron'+antiElectronDiscrMVA6_version+'_gbr_NoEleMatch_wGwoGSF_BL',
```

Comment:
this looks almost exactly like patTauDiscriminationByElectronRejectionMVA62018Raw, except for the veto.
If that's the intent, please change to 
```
patTauDiscriminationByElectronRejectionMVA62015Raw = patTauDiscriminationByElectronRejectionMVA62018Raw.clone(vetoEcalCracks = False)
```

Reply:
please note that the variable antiElectronDiscrMVA6_version changes. Therefore, the mva_Name parameters also have to be set, so this would not save much space and add additional dependency which makes future changes a bit more complicated. If you agree, I'd keep it like that.

Reply:
ok for me (partly also because it's not my signoff area).